include/drivers/pcie/msi.h

```diff
@@ -111,8 +112,8 @@ extern bool pcie_msi_enable(pcie_bdf_t bdf,
  * MSI capability IDs in the PCI configuration capability list.
  */
 
-#define PCIE_MSI_CAP_ID		0x05U
-#define PCIE_MSIX_CAP_ID	0x11U
+#define PCIE_MSI_CAP_ID		PCI_CAP_ID_MSI
+#define PCIE_MSIX_CAP_ID	PCI_CAP_ID_MSIX
```

Comment:
Can we just use `PCI_CAP_ID_MSI` instead of `PCIE_MSI_CAP_ID`.

Reply:
Obviously yes, I wanted to minimize changes but yes those defines should be dropped